Lambdas/GHHooks/EventHandler/Handlers/ReleaseMaker.swift

```diff
@@ -135,6 +135,7 @@ struct ReleaseMaker {
             if let (_, version) = SemanticVersion.fromGitHubTag(release.tagName) {
                 return (release, version)
             }
+            logger.warning("Could not parse tag", metadata: ["tag": .string(release.tagName)])
```

Comment:
I believe the preferred form for this is:
```suggestion
            logger.warning("Could not parse tag", metadata: ["tag": "\(release.tagName)"])
```

Reply:
I don't believe that tbh, I always use explicit .string when the thing is an string 😅

Shouldn't make much of a difference in the end but yeah that's what I've been going with